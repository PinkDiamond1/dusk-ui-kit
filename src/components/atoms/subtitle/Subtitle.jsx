/* @flow */
import * as React from 'react'

type Props = {
  className: string,
  children: React.Node,
}

const Subtitle = (props: Props): React.Element<*> => {
  const { className, children } = props
  const updatedClassName = `__duk-subtitle ${className}`
  return <h4 className={updatedClassName}>{children}</h4>
}

Subtitle.defaultProps = {
  className: '',
}

export default Subtitle
