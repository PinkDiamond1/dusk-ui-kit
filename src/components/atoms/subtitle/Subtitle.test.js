import React from 'react'
import renderer from 'react-test-renderer'

import Subtitle from './Subtitle'

it('test to see if the Subtitle renders correctly', () => {
  const tree = renderer.create(<Subtitle>Smoke Test</Subtitle>).toJSON()
  expect(tree).toMatchSnapshot()
})
