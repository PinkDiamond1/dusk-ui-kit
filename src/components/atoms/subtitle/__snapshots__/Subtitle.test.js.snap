// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`test to see if the Subtitle renders correctly 1`] = `
<h4
  className="__duk-subtitle "
>
  Smoke Test
</h4>
`;
